metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "praekeltfoundation-gluster",
+  "version": "0.0.1",
```

Comment:
I used `0.0.1` for "this doesn't actually work yet". Will bump to `0.1.0` for the first release.